layers/+tools/docker/packages.el

```diff
@@ -54,4 +54,6 @@
       (spacemacs/declare-prefix-for-mode 'dockerfile-mode
         "mc" "compile")
       (spacemacs/set-leader-keys-for-major-mode 'dockerfile-mode
-        "cb" 'dockerfile-build-buffer))))
+        "cb" 'dockerfile-build-buffer)
+      (spacemacs/set-leader-keys-for-major-mode 'dockerfile-mode
```

Comment:
This can be merged with the one above:

```emacs-lisp
(spacemacs/set-leader-keys-for-major-mode 'dockerfile-mode
         "cb" 'dockerfile-build-buffer
         "cB" 'dockerfile-build-no-cache-buffer)
```

Reply:
@sdwolfz yep, you are right, I have adjusted my code.